platform-hub-web/src/app/kubernetes-tokens/kubernetes-tokens-form.component.js

```diff
@@ -0,0 +1,103 @@
+export const KubernetesTokensFormComponent = {
+  bindings: {
+    transition: '<'
+  },
+  template: require('./kubernetes-tokens-form.html'),
+  controller: KubernetesTokensFormController
+};
+
+function KubernetesTokensFormController($state, hubApiService, logger, $stateParams) {
+  'ngInject';
+
+  const ctrl = this;
+
+  const userId = $stateParams.userId || (ctrl.transition && ctrl.transition.params().userId);
+  const cluster = ctrl.transition && ctrl.transition.params().cluster;
+
+  ctrl.loading = true;
+  ctrl.saving = false;
+  ctrl.isNew = true;
+  ctrl.tokenData = null;
+  ctrl.user = null;
+  ctrl.clusters = null;
+
+  ctrl.createOrUpdate = createOrUpdate;
+
+  init();
+
+  function init() {
+    logger.info('kube token form init');
+    ctrl.isNew = !cluster;
+
+    loadUser();
+
+    ctrl.clusters = [
+      {id: 'production', name: 'Production cluster'},
+      {id: 'development', name: 'Development cluster'},
+      {id: 'ci', name: 'CI cluster'},
+      {id: 'ops', name: 'OPS cluster'}
+    ];
```

Comment:
(as mentioned before) These should probably be driven by config to the backend, and then exposed in the API. Although, probably too much / complex data to be passed as config, so we could use the new `HashRecord` table here, expose it in an API, and either manually populate these in the db or provide an admin UI to edit these.

I can help with this if you think that's a good idea.